catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
 
 ## Description
 
-The unique UUID identifier for the media item.
+The unique UUID identifier for the media item. The `identifier` is generated
+when the item is first inserted into the main table.
 
 ## Object Shape
 
-UUID
-
-## Selection Criteria
-
-Created when the item is inserted into the main table.
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+[UUID](https://www.postgresql.org/docs/current/datatype-uuid.html#DATATYPE-UUID)
 
 # created_on
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
+The timestamp of when the media item was first added to Openverse catalog. This
+timestamp is generated when the item is first inserted into the main table.
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+```{note}
+This is _not_ the date when the item was first published on the source site.
+```
 
 # updated_on
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The timestamp of when the media item was last updated in Openverse catalog.
 
 # ingestion_type
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The way the media item was ingested into the Openverse catalog. `common_crawl`:
+data extracted from the Common Crawl dataset, when the Creative Commons search
+API was first created. `provider_api` data is extracted from various CC media
+provider APIs. `sql_bulk_load` data is extracted from the SQL data dumps.
 
 # provider
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
+The name of the provider of the media metadata. This is usually, but not always,
+the website that hosts the media item.
 
-## Normalization and Validation
+## Object Shape
 
-## Existing Data Inconsistencies
+This is a keyword for the provider, a string in a "snake_case" form.
 
 # source
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
+The name of the source of the media item.It can be a collection on a provider
+site, or the provider itself.
 
-## Normalization and Validation
+## Object Shape
 
-## Existing Data Inconsistencies
+This is a keyword for the source, a string in a "snake_case" form.
 
 # foreign_identifier
 
 ## Description
 
-## Object Shape
+The unique identifier for the media item on the source site.
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+Ideally, this identifier should be usable to generate the URL to the media item
+on the source site.
```

Comment:
I don't think this is necessarily accurate, especially since we generally don't construct any reference URLs ourselves. I might recommend changing this to something like: "This is usually the 'ID' value for an item that the upstream provider supplies".